PWGCF/FEMTOSCOPY/FemtoDream/AliAnalysisTaskNanoPt.h

```diff
@@ -0,0 +1,97 @@
+#ifndef AliAnalysisTaskNanoPt_H
+#define AliAnalysisTaskNanoPt_H
+
+#include "AliAnalysisTaskSE.h"
+#include "AliConvEventCuts.h"
+#include "AliEventCuts.h"
+#include "AliFemtoDreamCollConfig.h"
+#include "AliFemtoDreamEventCuts.h"
+#include "AliFemtoDreamPairCleaner.h"
+#include "AliFemtoDreamPartCollection.h"
+#include "AliFemtoDreamTrack.h"
+#include "AliFemtoDreamTrackCuts.h"
+#include "AliFemtoDreamControlSample.h"
+#include "AliMCEvent.h"
+#include "AliStack.h"
+#include "TChain.h"
+class AliVParticle;
+class AliVTrack;
+
+class AliAnalysisTaskNanoPt : public AliAnalysisTaskSE {
+ public:
+  AliAnalysisTaskNanoPt();
+  AliAnalysisTaskNanoPt(const char *name, const bool isMC);
+  virtual ~AliAnalysisTaskNanoPt();
+  Float_t GetMass2sq(AliFemtoDreamTrack *track);
```

Comment:
I would mark that function as const
https://docs.microsoft.com/en-us/cpp/cpp/const-cpp?view=vs-2019#const-member-functions
(which is a nice documentation btw)